lms/templates/instructor/instructor_dashboard_2/cohorts.underscore

```diff
@@ -47,5 +47,12 @@
                 ) %>
             </p>
         </div>
+
+        <hr class="divider divider-lv1" />
+
+        <!-- Discussion Topics. -->
+        <a class="toggle-cohort-management-discussions" href="#cohort-discussions-management"><%- gettext('Assign cohorts to discussion topics') %></a>
```

Comment:
See at line 3 of lms/templates/instructor/instructor_dashboard_2/cohort-discussions.underscore about behavior on clicking heading. Suggested new heading might be "Specify whether discussion topics are divided by cohort". @explorerleslie @marcotuts your thoughts?